### PR TITLE
Loading package via VCS instead of path

### PR DESCRIPTION
**Pull request by Mcbeany**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,12 +27,12 @@
     ],
     "repositories": [
         {
-            "type": "path",
-            "url": "./lib/Commando-master/"
+            "type": "vcs",
+            "url": "https://github.com/jojoe77777/FormAPI.git"
```
Comment (MintoD) [inline composer.json:31]: Use https://github.com/PocketMine-MP-VN-Group/FormAPI

File: composer.json [hunk 2]
```diff
@@ -27,12 +27,12 @@
...
         },
         {
-            "type": "path",
-            "url": "./lib/FormAPI-master/"
+            "type": "vcs",
+            "url": "https://github.com/CortexPE/Commando.git"
```
Comment (MintoD) [inline composer.json:35]: Commando by CortexPE not supported PocketMine latest version. Use [Commando](https://github.com/Paroxity/Commando) by Paroxity

**Review by MintoD — CHANGES_REQUESTED** [review]

File: composer.json [hunk 3]
```diff
@@ -27,12 +27,12 @@
...
         {
-            "type": "path",
-            "url": "./lib/Commando-master/"
+            "type": "vcs",
+            "url": "https://github.com/jojoe77777/FormAPI.git"
         },
         {
-            "type": "path",
-            "url": "./lib/FormAPI-master/"
+            "type": "vcs",
```
Comment (MintoD) [inline composer.json:34]: Using Commando on Packagist

**Review by MintoD — CHANGES_REQUESTED** [review]

**Review by Mcbeany — COMMENTED** [review]
ok looks good

**Pull request closed without merging**